servers/rendering/rendering_device.cpp

```diff
@@ -5962,22 +6122,24 @@ void RenderingDevice::_end_frame() {
 	driver->end_segment();
 }
 
-void RenderingDevice::_execute_frame(bool p_present) {
-	// Check whether this frame should present the swap chains and in which queue.
-	const bool frame_can_present = p_present && !frames[frame].swap_chains_to_present.is_empty();
-	const bool separate_present_queue = main_queue != present_queue;
-	thread_local LocalVector<RDD::SwapChainID> swap_chains;
-	swap_chains.clear();
-
-	// Execute command buffers and use semaphores to wait on the execution of the previous one. Normally there's only one command buffer,
-	// but driver workarounds can force situations where there'll be more.
+void RenderingDevice::execute_chained_cmds(const bool present_swap_chain, const RDD::FenceID fence,
```

Comment:
```suggestion
void RenderingDevice::execute_chained_cmds(bool p_present_swap_chain, RDD::FenceID p_fence,
```

Feel free to read the `const` (I guess that ship already sailed), though.